src/app/courses/add-courses/courses-step.component.ts

```diff
@@ -79,6 +79,11 @@ export class CoursesStepComponent implements OnDestroy {
     };
   }
 
+  removeResource(position: number) {
+    console.log(position);
+    this.steps.slice(position, 1);
```

Comment:
`this.steps[this.activeStepIndex].resources.slice(position, 1);`

Reply:
@paulbert cant remove the item from the steps though code updated. Please review this.